PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/serviceexplorer/azure/database/actions/OpenInBrowserAction.kt

```diff
@@ -49,4 +50,6 @@ abstract class OpenInBrowserAction(private val subscriptionId: String, private v
             throw RuntimeException(message)
         }
     }
+
+    override fun getIconPath(): String = "OpenInBrowser.svg"
```

Comment:
Not sure if it matters, but will this still work with the "upstream"? I think they are using .png?

Reply:
Do you mean all missing icons that we replaced in the fork? I think we can ignore it for now. It will not in the same way like many other icons where we have just changed `.png` to `.svg` in `Utils`. I think we can handle all this properly building an our own IconManager that will handle icons mapping in future if needed.

Reply:
Cool, in that case LGTM

Reply:
FYI, I will merge this after https://github.com/JetBrains/azure-tools-for-intellij/pull/335